sw-precache-config.js

```diff
@@ -1,11 +1,9 @@
 module.exports = {
   staticFileGlobs: [
     '/manifest.json',
```

Comment:
Should not routes be relative?

Like the PSK: https://github.com/PolymerElements/polymer-starter-kit/pull/1008

Reply:
It doesn't seem to make a difference, as `polymer build` will remove the leading slash (so it's always relative to the specific build, like `es6-bundled/`).